### ISSUE TITLE
Feature Request: keyboard translation options

### ISSUE THREAD

**Issue by Dworf** [bitgapp/Cursr#37]
I'm testing and playing with version v1.5.0-beta.1:

When controlling, for example, WIN from MAC, it would be great to have keyboard translation options to select for example:

CTRL → Control / Option / Command
Option → CTRL / ALT / WIN

Etc...

For example, I'm using a foreign keyboard and on MAC I have:

On MAC: OPTION + Q produces: \ ASCII key
On Windows (with Windows keyboard): if I press the ALT + Q key I get the right: \ ASCII key

now using Cursr:
On Windows (when controlling through Cursr on MAC) if I press OPTION + Q key I get a menu popup from the app running...

**Comment by pltrant** [comment]
Yes, the ability to remap Ctrl/Win/Alt (Windows) to Ctrl/Option/Command (Mac) is highly desirable.

**Comment by joerohde** [comment]
Yes.  Difficult (for me) to use my Windows box to control my mac unless I can swap Alt/Win so it maps to Cmd/Option to match physical layouts.

**Comment by ragauskl** [comment]
It is planned and will be added at some point, however for now there are a few higher priority tasks, so likely won't be sooner than in next couple months
